entrypoint.sh

```diff
@@ -52,7 +52,7 @@ if [ ! -z "${variables}" ]; then
 fi
 
 if [ ! -z "$file" ]; then
-params="${params} --filename $file"
```

Comment:
Do we need to update the CLI version too?

Reply:
In theory, that would be done by the Okteto bot once the new CLI version is released. Now that you mention, doing that, would leave a github action version "broken", at least until we merge this and we would have to overwrite the version tag to include this. To avoid that, I think we can change it to use the short notation `-f`. As that is transparent for the users. Doing that, we could merge this PR now instead of waiting for the CLI release and overwrite the tag version.

What do you think @jLopezbarb ?